test/conformance/kuttl/mutate/clusterpolicy/standard/existing/target-context/policy.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: update-targets
+spec:
+  background: false
+  rules:
+  - name: update-targets
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+    context:
+    - name: triggerContent
+      variable:
+        jmesPath: request.object.data.content
+    preconditions:
+      all:
+      - key: "{{ request.object.metadata.name }}"
+        operator: Equals
+        value: trigger
+    mutate:
+      targets:
+        - apiVersion: v1
+          kind: ConfigMap
+          namespace: "{{ request.object.metadata.namespace }}"
+          name: target*
```

Comment:
I've never seen a wildcard used here before, but you're not exactly testing it if there's only one target and it's named `target`. I guess it would fail if it didn't work. Does it make sense to create a second existing ConfigMap named something like `targetextra` and assert at the end that it also got mutated?

Reply:
Wildcard is supported in both name and namespace.
If I don’t specify a name there’s a race and it ends up mutating the trigger resource too ;)

Reply:
> Does it make sense to create a second existing ConfigMap named something like targetextra and assert at the end that it also got mutated?

Not really the scope of this test IMHO. Maybe I should just use `target` ?

Reply:
Right, not the scope of this test. Leaving what you have is fine, it was just commentary.

>If I don’t specify a name there’s a race and it ends up mutating the trigger resource too ;)

Good to know. We always encourage using the `name` field.

Reply:
Maybe we should exclude the trigger resource from the list of target resources ?
But I fear there are use cases for that too.

Reply:
Maybe we punt for now. Technically a trigger resource and a target resource are mutually exclusive because a trigger has to come in on the admission chain whereas a target does not. We may not want to stir the pot any more right now until we have to.

Reply:
>Maybe we should exclude the trigger resource from the list of target resources ?

Agreed. We had discussed this to avoid endless loop for mutate existing, same for generate.

>But I fear there are use cases for that too.

A mutate rule either mutates incoming objects or the target resource (other than the trigger).

>We may not want to stir the pot any more right now until we have to.

+1 :)